src/components/10-atoms/input-text/react/DemoInputTextonKeyDown.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import AXAInputText from './AXAInputText';
+
+const DemoInputTextonKeyDown = () => {
+  return (
+    <div>
+      <p>
+        <span>Last key pressed: </span>
+        <span id="inputtext-react-testoutput" />
+      </p>
+      <AXAInputText
+        name="onkeydown"
+        id="inputtext-react-inputfield-on-key-pressed"
+        onKeyDown={e => {
+          document.querySelector('#inputtext-react-testoutput').innerHTML =
```

Comment:
Don't use innerHTML whenever a safer alternative exists - here textContent seems enough, no?

Reply:
I changed it to textContent. Thx.